sandbox/registration-processor-mz.properties

```diff
@@ -484,6 +484,12 @@ mosip.regproc.securezone.notification.eventbus.kafka.poll.frequency=100
 mosip.regproc.securezone.notification.eventbus.kafka.group.id=securezone-notification-stage
 mosip.regproc.securezone.notification.message.expiry-time-limit=${mosip.regproc.common.stage.message.expiry-time-limit}
 
+mosip.regproc.securezone.notification.server.port=8090
+mosip.regproc.securezone.notification.server.servlet.path=/registrationprocessor/v1/securezone
+mosip.regproc.securezone.notification.eventbus.port=5712
+
+mosip.regproc.message.tag.loading.disable=true
```

Comment:
As discussed, this property should be specific to each stage, approving the changes for now please take care of this in next PR